backend/src/pong/game/game.service.ts

```diff
@@ -1,15 +1,15 @@
 import { Injectable } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 import { AbstractService } from '../../shared/abstract.service';
-import { Repository } from 'typeorm';
+import { InsertResult, Repository } from 'typeorm';
```

Comment:
not sure if this was added because of the sloppy merge with main... but `insertResult` isn't used

Reply:
Nois!